api/openapi.yaml

```diff
@@ -8023,15 +8023,24 @@ components:
         links:
           key:
             href: /some-resource/d98d621a-23da-49fe-af96-6be709abb718
+        included:
+        - ""
+        - ""
       properties:
         data:
           $ref: '#/components/schemas/WebhookEventData'
+        included:
+          items:
+            anyOf:
+            - $ref: '#/components/schemas/WebhookEventSceneIncludedData'
+          type: array
```

Comment:
Shoot, I should have noticed this earlier. This should really be a `oneOf` where the discriminator is the `type` in the included resource object. `anyOf` means that the payload can match more than one schema, which doesn't seem correct in this scenario.

You can find an example of OpenAPI discriminator types here: https://github.com/Vertexvis/vertex-api/blob/0bf48fcacae1177bcf96c26e625e3fc6e784f5cc/src/universal/schemas/WebhookEventData.yml

Reply:
Ah, that makes sense. I was thinking of the `anyOf` as more fitting given the array typing, but the `oneOf` also makes sense. I'll push a another vertex-api PR